tests/unit-tests/test_gnn.py

```diff
@@ -2471,6 +2512,448 @@ def check_call_gen_embs(skip_last_self_loop):
     skip_self_loop = False
     gen_emb_for_nfeat_reconstruct(model, check_call_gen_embs)
 
+def test_edge_model_inference_with_edge_feats():
+    """ Test the ``do_mini_batch_inference`` method with edge features.
+    
+    Currently only support edge features in mini-batch inference. So, will not compare
+    the results between full-graph inference and mini-batch inference.
+    """
+    # initialize the torch distributed environment
+    th.distributed.init_process_group(backend='gloo',
+                                      init_method='tcp://127.0.0.1:23456',
+                                      rank=0,
+                                      world_size=1)
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        # get the test dummy distributed graph
+        _, part_config = generate_dummy_dist_graph(tmpdirname)
+        ep_data = GSgnnData(part_config=part_config,
+                            node_feat_field="feat",
+                            edge_feat_field="feat")
+
+    # Test case 0: normal case, model uses edge feature, and provides edge feature
+    #              for mini-batch inference
```

Comment:
There can be another case: model uses edge feature feat0 but provides edge feature feat1 for mini-batch.
what will happen in this case?

Reply:
Add another test case, this will trigger an assert error. In addition, this case has been tested in `test_embed.py` line #281.